opm/autodiff/MissingFeatures.cpp

```diff
@@ -55,6 +89,14 @@ namespace MissingFeatures {
             "VAPPARS", "VISCREF", "WATVISCT",
             "WPAVE", "WPIMULT", "WPITAB", "WTEMP",
             "WTEST", "WTRACER", "ZIPPY2" };
+        std::multimap<std::string, PartiallySupported<std::string> > string_options;
+        std::multimap<std::string, PartiallySupported<int> > int_options;
+        addUnsupported<ParserKeywords::COMPORD, ParserKeywords::COMPORD::ORDER_TYPE, std::string>(string_options , "TRACK");
```

Comment:
>perhaps there are many more unsupported than supported options for this keyword?

This is the main reason why we check supported options, I will rename the function.